runtime/src/lib.rs

```diff
@@ -289,7 +289,8 @@ impl pallet_signal::Config for Runtime {
 }
 
 impl pallet_fennel_identity::Config for Runtime {
-    type Event = Event;
+  type Event = Event;
+  type WeightInfo = ();
```

Comment:
@Romulus10 I think if you change the trust pallet's Config impl for Runtime to use () instead of the substrate object, then you won't need the additional impl in weights.rs

Reply:
Hm. That's a thought. It even compiles and tests properly. 

I wonder why it works that way everywhere else then?

Reply:
Like why specifically implement for WeightInfo, you know?